_includes/navbar.ext

```diff
@@ -31,6 +31,7 @@
                   <li><a href="/technical_notes.html">Technical Notes</a></li>
                   <li><a href="/newsletter.html">Newsletter</a></li>
                   <li><a href="/organization/minutes.html">Meeting Notes</a></li>
+                  <li><a href="/calendar.html">Community Calendar Editing</a></li>
```

Comment:
Why not to add this menu  to 'Get  Involved' rather than 'Communication'?

Reply:
I'm ok with that. As *Get Involved* is a single page link, do you think just add calendar editing as part of that page? It's short enough that a separate page is perhaps unnecessary.